release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -168,13 +168,22 @@ If you are upgrading your cluster to {product-title} 4.6, old tokens from {produ
 [id="ocp-4-6-machine-api"]
 === Machine API
 
+[id="ocp-4-6-machine-api-multiple-block-device-mappings"]
+==== Support for multiple block device mappings
+
+The Machine API now supports multiple block device mappings. Logs, data in empty directory pods, and docker images are now stored in separate block devices from the root device on a machine.
```

Comment:
> Logs, data in empty directory pods, and docker images are now stored in separate block devices from the root device on a machine.

This isn't always the case (eg if there's only one block device given), I'm not sure if it's ever the case, I'm not sure entirely what the user case is for multiple block devices though. @alexander-demichev Do you have anything to add here?

Reply:
@JoelSpeed @alexander-demichev Would it be more appropriate to say that a user _can_ use multiple block devices?

> You can now store logs, data in empty directory pods, and docker images in separate block devices from the root device on a machine.

Reply:
I think that's more appropriate yes, though I'd still like @alexander-demichev to confirm